generate.sh

```diff
@@ -128,33 +131,33 @@ if [ "${AUTOAPI_JAVA:-false}" == "true" ]; then
 fi
 
 if [ ! -f index.rst ]; then
-  echo -e "No index.rst found. Auto generating...\n"
+  >&2 echo -e "No index.rst found. Auto generating...\n" | tee -a ${LOGFILE}
   python $BASE/modules/scripts/genindex.py $ROOT_DIR
   if [ "${DEBUG:-false}" == "true" ]; then
```

Comment:
With this PR, I don't think `DEBUG` has any use now. Remove it completely i.e from the script, webUI, config.py